tests/runtime/test_grpc_server.py

```diff
@@ -280,7 +283,7 @@ def test_train_fake_module_does_not_change_another_instance_model_of_block(
 
     # make sure the trained model can run inference, and the batch size 100 was used
     predict_request = sample_inference_service.messages.OtherTaskRequest(
-        sample_input=HAPPY_PATH_INPUT
+        sample_inputsampleinputtype=HAPPY_PATH_INPUT
```

Comment:
🤢🤢🤢 oof this name...

I think I'd expect a `_` between the oneof name and the type name, should we change that?

Reply:
Yeah I would honestly like that too, but that's set in `py-to-proto` right?

Reply:
Feel free to submit a PR! https://github.com/IBM/py-to-proto/blob/main/py_to_proto/dataclass_to_proto.py#L257

Reply:
Ookay, so this is a breaking change and will be fixed by a major bump for py-to-proto then!  Let's keep it for a later release ?

Reply:
Sure!